tensorflow/lite/micro/kernels/add_test.cc

```diff
@@ -362,4 +362,132 @@ TF_LITE_MICRO_TEST(QuantizedAddWithMixedBroadcastInt8) {
   }
 }
 
+TF_LITE_MICRO_TEST(QuantizedAddNoActivationInt16) {
+  const float scales[] = {0.25, 0.5, 1.0};
+  const int zero_points[] = {0, 0, 0};
+  const int output_dims_count = 4;
+  int inout_shape[] = {4, 1, 2, 2, 1};
+  const float input1_values[] = {-2.01, -1.01, -0.01, 0.98};
+  const float input2_values[] = {1.01, 2.01, 3.01, 4.02};
+  const float golden_values[] = {-1, 1, 3, 5};
+
+  int16_t input1_quantized[output_dims_count];
+  int16_t input2_quantized[output_dims_count];
+  int16_t golden_quantized[output_dims_count];
+  int16_t output[output_dims_count];
+
+  tflite::testing::TestAddQuantized(
+      inout_shape, input1_values, input1_quantized, scales[0], zero_points[0],
+      inout_shape, input2_values, input2_quantized, scales[1], zero_points[1],
+      inout_shape, golden_values, golden_quantized, scales[2], zero_points[2],
+      kTfLiteActNone, output);
+}
+
+TF_LITE_MICRO_TEST(QuantizedAddActivationRelu1Int16) {
+  const float scales[] = {0.25, 0.5, 1.0};
+  const int zero_points[] = {0, 0, 0};
+  const int output_dims_count = 4;
+  int inout_shape[] = {4, 1, 2, 2, 1};
+  const float input1_values[] = {-2.01, -1.01, -0.01, 0.98};
+  const float input2_values[] = {1.01, 1.99, 2.99, 4.02};
+  const float golden_values[] = {-1, 1, 1, 1};
+
+  int16_t input1_quantized[output_dims_count];
+  int16_t input2_quantized[output_dims_count];
+  int16_t golden_quantized[output_dims_count];
+  int16_t output[output_dims_count];
+
+  tflite::testing::TestAddQuantized(
+      inout_shape, input1_values, input1_quantized, scales[0], zero_points[0],
+      inout_shape, input2_values, input2_quantized, scales[1], zero_points[1],
+      inout_shape, golden_values, golden_quantized, scales[2], zero_points[2],
+      kTfLiteActReluN1To1, output);
+}
+
+TF_LITE_MICRO_TEST(QuantizedAddVariousInputShapesInt16) {
+  const float scales[] = {0.1, 0.05, 0.1};
+  const int zero_points[] = {0, 0, 0};
+  const int output_dims_count = 6;
```

Comment:
constexpr, and move closer to first use?

Reply:
Done.